x/stake/keeper/validator.go

```diff
@@ -317,8 +388,9 @@ func (k Keeper) updateValidatorPower(ctx sdk.Context, oldFound bool, oldValidato
 //
 // nolint: gocyclo
 // TODO: Remove the above golint
-func (k Keeper) UpdateBondedValidators(ctx sdk.Context,
-	affectedValidator types.Validator) (updatedVal types.Validator, updated bool) {
+func (k Keeper) UpdateBondedValidators(
+	ctx sdk.Context, affectedValidator types.Validator,
+) (updatedVal types.Validator, updated bool) {
```

Comment:
better indentation: 

```golang
func (k Keeper) UpdateBondedValidators(
    ctx sdk.Context, affectedValidator types.Validator) (
    updatedVal types.Validator, updated bool) {
```

Reply:
Tbh, I don't necessarily agree here because it's hard for me to immediately see the return values (and their potential names) due to the location of the opening parenthesis. (e.g. immediate glance seems to show it has no returns).

But this is your baby and I'd like to keep in-line with what you've written and the team's standards, so I can update it.